node/pkg/p2p/p2p.go

```diff
@@ -74,11 +80,6 @@ var (
 			Name: "wormhole_p2p_drops",
 			Help: "Total number of messages that were dropped by libp2p",
 		})
```

Comment:
Do you want to add a gauge for observations in a batch? :shrug:

Reply:
Meh.